worlds/tunic/__init__.py

```diff
@@ -440,7 +456,7 @@ def stage_pre_fill(cls, multiworld: MultiWorld) -> None:
     def create_regions(self) -> None:
         self.tunic_portal_pairs = {}
         self.er_portal_hints = {}
-        self.ability_unlocks = randomize_ability_unlocks(self.random, self.options)
+        self.ability_unlocks = randomize_ability_unlocks(self)
 
         # stuff for universal tracker support, can be ignored for standard gen
         if self.using_ut:
```

Comment:
```suggestion
        if self.using_ut and self.options.hexagon_quest_ability_type == "hexagons":
```

Reply:
What exactly does this achieve? I might be missing something UT specific but wouldn't those have a value regardless

Reply:
Oh right, I forgot those still get the 0 value, nvm